src/layer/index.ts

```diff
@@ -710,6 +710,7 @@ export class ManagedUserLayer extends RefCounted {
   }
 
   visible = true;
+  codeVisible = new TrackableBoolean(true, true);
```

Comment:
I don't think there is a reason to make this a ManagedUserLayer property --- instead it should be a property of the specific UserLayer types that have a shader widget.